src/views/ReadArticle.js

```diff
@@ -305,8 +309,24 @@ export class Article extends Component {
                 <img className="share-icon" src={thumbsUp} alt="logo" />
                 <img className="share-icon" src={dislikeIcon} alt="logo" />
                 <img className="share-icon" src={bookmarkIcon} alt="logo" />
-                <img className="share-icon" src={facebookIcon} alt="logo" />
-                <img className="share-icon" src={twitterIcon} alt="logo" />
+                <a
+                  className="share-icon"
+                  href={`https://www.facebook.com/sharer/sharer.php?&u=${currentUrl}`}
+                >
+                  <img className="share-icon" src={facebookIcon} alt="logo" />
+                </a>
```

Comment:
@Musigwa how about if you create a single component, and then use that single component to create multiple links for different platforms?

Reply:
What will be the difference from this I did?

Reply:
Each button has its own link to redirect the user to, So, can you elaborate the reason why I should do that?